src/Sylius/Bundle/UserBundle/spec/Authentication/AuthenticationFailureHandlerSpec.php

```diff
@@ -42,9 +48,14 @@ function it_is_a_authentication_failure_handler(): void
     function it_returns_json_response_if_request_is_xml_based(
         Request $request,
         AuthenticationException $authenticationException,
+        TranslatorInterface $translator,
     ): void {
         $request->isXmlHttpRequest()->willReturn(true);
+        $request->getLocale()->willReturn('fr');
         $authenticationException->getMessageKey()->willReturn('Invalid credentials.');
+        $translator->trans('Invalid credentials.', [], 'security', 'fr')
+            ->willReturn('Identifiants invalides.')
+            ->shouldBeCalled();
```

Comment:
I recommend running `vendor/bin/ecs` before pushing the PR.
```suggestion
            ->shouldBeCalled()
        ;
```